src/main/kotlin/io/github/moulberry/notenoughupdates/miscfeatures/profileviewer/SacksPage.kt

```diff
@@ -61,7 +69,27 @@ class SacksPage(pvInstance: GuiProfileViewer) : GuiProfileViewerPage(pvInstance)
     private val sackGridYSize = 41
     private val itemIconSize = 20
 
-    private val sackContents = mutableMapOf<String, SackInfo>()
+    // Lazy initialisation to allow for guiLeft and guiTop to be initialized first
+    private val priceSourceButtonRect by lazy { Rectangle(guiLeft + 54, guiTop + 155, 20, 20) }
```

Comment:
Technically this breaks if the window gets resized. Then again, we don't have any mechanism for invalidating these rects, so this will be fine.